source/common/config/grpc_mux_impl.h

```diff
@@ -171,6 +174,9 @@ class GrpcMuxImpl : public GrpcMux,
   const LocalInfo::LocalInfo& local_info_;
   const bool skip_subsequent_node_;
   CustomConfigValidatorsPtr config_validators_;
+  // Not owned; can be nullptr.
+  XdsResourcesDelegate* xds_resources_delegate_;
```

Comment:
Generally Envoy leans towards OptRef for optional references, rather than pointers.

Reply:
Switched to OptRef